cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -585,4 +596,51 @@ def test_custom_button_dialog(button_group):
     Bugzilla:
         1574774
     """
-    pass
+    group, obj_type = button_group
+    with appliance.context.use(ViaUI):
+        button = group.buttons.create(
+            text="btn_{}".format(fauxfactory.gen_alphanumeric(3)),
+            hover="btn_hover_{}".format(fauxfactory.gen_alphanumeric(3)),
+            dialog=dialog,
+            system="Request",
+            request="InspectMe",
+        )
+        request.addfinalizer(button.delete_if_exists)
+
+    with appliance.context.use(context):
+        navigate_to = ssui_nav if context is ViaSSUI else ui_nav
+
+        obj = objects[obj_type]["Details"][0]
+        dest_name = objects[obj_type]["Details"][1]
+        view = navigate_to(obj, dest_name)
+        custom_button_group = Dropdown(view, group.text)
+        assert custom_button_group.has_item(button.text)
+
+        # Clear the automation log
+        assert appliance.ssh_client.run_command(
+            'echo -n "" > /var/www/miq/vmdb/log/automation.log'
```

Comment:
QUESTION: if test undertakes this step, it will screw up automation.log artifact. I'd try stear clear of such behaviour. What is it for ? Would you like to speed up your test ?

Reply:
Yes, I would like to speed up.... let me know  suggestions

Reply:
@izapolsk I created one card `RHCFQE-11593` to remove use of `log_request_check` while request check. For now, I would like to go with current custom button implementation.

Reply:
okay, please don't wait for long with new implementation.